postreise/plot/plot_utilization_map.py

```diff
@@ -35,92 +20,109 @@ def map_risk_bind(
     vmin=None,
     vmax=None,
     is_website=False,
+    palette=None,
+    all_branch_scale_factor=5e-4,
+    all_branch_min_width=0.2,
+    select_branch_scale_factor=1e-3,
+    select_branch_min_width=2,
+    figsize=(1400, 800),
+    show_color_bar=True,
+    plot_states_kwargs=None,
 ):
     """Makes map showing risk or binding incidents on US states map.
 
     :param str risk_or_bind: specify plotting "risk" or "bind"
     :param pandas.DataFrame congestion_stats: data frame as returned by
         :func:`postreise.analyze.transmission.utilization.generate_cong_stats`.
     :param pandas.DataFrame branch: branch data frame.
-    :param dict us_states_dat: dictionary of state border lats/lons. If None, get
-        from :func:`postreise.plot.plot_states.get_state_borders`.
     :param int/float vmin: minimum value for color range. If None, use data minimum.
     :param int/float vmax: maximum value for color range. If None, use data maximum.
     :param bool is_website: changes text/legend formatting to look better on the website
-    :return:  -- map of lines with risk and bind incidents color coded
+    :param iterable palette: sequence of colors used for color range, passed as
+        `palette` kwarg to :func:`bokeh.transform.linear_cmap`.
+        If None, default to `postreise.plot.colors.traffic_palette`.
+    :param int/float all_branch_scale_factor: scale factor for unhighlighted branches.
+    :param int/float all_branch_min_width: minimum width for unhighlighted branches.
+    :param int/float select_branch_scale_factor: scale factor for highlighted branches.
+    :param int/float select_branch_min_width: minimum width for highlighted branches.
+    :param tuple(int, int) figsize: size of the bokeh figure (in pixels).
+    :param bool show_color_bar: whether to render the color bar on the figure.
+    :param dict plot_states_kwargs: keyword arguments to be passed to
+        :func:`postreise.plot.plot_states.plot_states`.
+    :return: (*bokeh.plotting.figure*) -- map of lines with risk and bind incidents
+        color coded.
     """
```

Comment:
should we raise some errors for the parameters, in particular for `risk_or_bind`?

Reply:
Done.

Reply:
We should add the error raised in the docstring

Reply:
Done.